bin/metrics-springboot.rb

```diff
@@ -44,20 +49,46 @@
 require 'uri'
 
 class SpringBootMetrics < Sensu::Plugin::Metric::CLI::Graphite
+  option :url,
+         short: '-u URL',
+         long: '--url URL',
+         description: 'The url for your app metrics',
+         required: false
+
   option :host,
          short: '-h HOST',
          long: '--host HOST',
          description: 'Your spring boot actuator endpoint',
-         required: true,
+         required: false,
          default: 'localhost'
 
   option :port,
          short: '-P PORT',
          long: '--port PORT',
          description: 'Your app port',
-         required: true,
+         required: false,
          default: 8080
 
+  option :path,
+         short: '-e PATH',
+         long: '--path PATH',
+         description: 'Metrics endpoint path',
+         required: false,
+         default: '/metrics'
+
+  option :ssl,
```

Comment:
rather than creating an argument called `ssl` I think it makes more sense to create one called protocol, that way you do not have to make conditional checks all over the place you just specify the protocol option and then in the option definition you default it to `http` (to keep backwards compatibility) and specify that the acceptable options are http and https. For example:
```ruby
option :protocol,
           long: '--protocol $PROTO',
           description: 'The protocol used to make requests',
          in: %[http https],
          default: http
```